src/style.css

```diff
@@ -0,0 +1,81 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./Fonts/Avenir/Avenir.ttf) format("truetype");
+  font-weight: normal;
+  font-style: normal;
+}
+
+.search-form::after {
+  position: absolute;
+  top: 25px;
+  left: 26px;
+  content: "";
+  width: 19px;
+  height: 19px;
+  background-image: url(./images/Search.svg);
+}
+
+.search-form_small {
+  position: relative;
+}
+
+.search-form_small::after {
+  top: 15px;
+  left: 13px;
+  width: 11px;
+  height: 11px;
+  background-size: cover;
+}
+
+.search-form__input {
+  width: 100%;
+  height: 70px;
+  border: 1px solid #e1e7ed;
+  -webkit-box-sizing: border-box;
+          box-sizing: border-box;
+  -webkit-box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+          box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  font-family: Avenir, sans-serif;
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
+  padding-left: 62px;
+  position: relative;
+}
+
+.search-form__input::-webkit-input-placeholder {
+  color: #3d4e61;
+}
+
+.search-form__input:-ms-input-placeholder {
+  color: #3d4e61;
+}
+
```

Comment:
why you duplicate this class again?
`.search-form__input:-ms-input-placeholder` vs `.search-form__input::-ms-input-placeholder`